tardis/conftest.py

```diff
@@ -56,6 +56,11 @@ def pytest_addoption(parser):
 
 
 def pytest_configure(config):
```

Comment:
Does py.test require `pytest_configure` to be in the root conftest? If that's not the case, we should everything related to slow_tests to the respective directory.